opendistro-for-elasticsearch/README.md

```diff
@@ -0,0 +1,18 @@
+# elasticsearch-odfe
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Service package
+
+## Usage
+
+```bash
+hab pkg install core/elasticsearch-odfe
```

Comment:
These should be updated to match the name change :smile:

Reply:
doh! Fixed.